libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -511,18 +529,44 @@ class MapboxNavigation(
          * Use [NavigationOptions.toBuilder] to easily customize selected options.
          */
         @JvmStatic
-        fun defaultNavigationOptions(context: Context): NavigationOptions {
-            return NavigationOptions.Builder(
-                NONE_SPECIFIED,
-                ROUNDING_INCREMENT_FIFTY,
-                DEFAULT_NAVIGATOR_POLLING_DELAY,
-                MapboxDistanceFormatter(
-                    context.applicationContext,
-                    null,
-                    UNDEFINED,
-                    ROUNDING_INCREMENT_FIFTY
+        fun defaultNavigationOptions(context: Context, accessToken: String?): NavigationOptions {
+            val tilesUri = URI("https://api-routing-tiles-staging.tilestream.net")
+            val tilesVersion = "2019_04_13-00_00_11"
+            return NavigationOptions.Builder()
+                .timeFormatType(NONE_SPECIFIED)
+                .roundingIncrement(ROUNDING_INCREMENT_FIFTY)
+                .navigatorPollingDelay(DEFAULT_NAVIGATOR_POLLING_DELAY)
+                .distanceFormatter(
+                    MapboxDistanceFormatter(
+                        context.applicationContext,
+                        null,
+                        UNDEFINED,
+                        ROUNDING_INCREMENT_FIFTY
+                    )
                 )
-            ).build()
+                .onboardRouterConfig(
+                    MapboxOnboardRouterConfig(
+                        obtainOfflineTilesDir(tilesUri.host, tilesVersion),
+                        null,
+                        null,
+                        2,
+                        Endpoint(
+                            tilesUri.toString(),
+                            tilesVersion,
+                            accessToken ?: "",
+                            "MapboxNavigationNative"
+                        )
+                    )
+                ).build()
+        }
+
+        private fun obtainOfflineTilesDir(host: String, version: String): String {
+            val offlineDir = Environment.getExternalStoragePublicDirectory("Offline")
+            if (!offlineDir.exists()) {
+                Log.d("MapboxNavigation", "Offline directory does not exist")
+                offlineDir.mkdirs()
+            }
+            return File(offlineDir, "$host/$version").absolutePath
```

Comment:
What's the plan to manage these files? I've heard they're ~300MB? And then whenever we change the router config the old one isn't cleaned up. If a customer integrates the SDK this big file will be stored. They update the app or the configuration, the old one is still there.

Reply:
That's another Nav Native feature that remove's the tiles. We can follow up in a separate PR.

Reply:
For what it's worth, put some thought into it and preference was to put the responsibility on the apps that integrate the SDK until that feature is verified